solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/List/ListView.swift

```diff
@@ -3,15 +3,19 @@
 import UIKit
 
 final class ListView: UIView {
-  private let listViewCellIdentifier = "ListViewCellIdentifier"
+  private let listViewCellIdentifier = "RepositoryCellView"
 
-  private var listItems: [String] = []
+  private var listItems: [RepositoryCellViewConfiguration] = [
+    RepositoryCellViewConfiguration(repositoryName: "hereminders-ios", repositoryOwnerName: "rdgborges"),
```

Comment:
We shouldn't have these hardcoded configurations in the production code. Can we remove them?